custom_registration/models.py

```diff
@@ -111,3 +115,23 @@ def send_new_user_email(user):
     )
     email.attach_alternative(html_body, 'text/html')
     email.send()
+
+
+def eligible_team_by_email_domain(user):
```

Comment:
@s-kennedy is there a reason why you added this here rather than in `studygroups/models/team.py`?